librz/analysis/p/analysis_bf.c

```diff
@@ -7,35 +8,446 @@
 #include <rz_asm.h>
 #include <rz_analysis.h>
 
-static size_t countChar(const ut8 *buf, int len, char ch) {
-	int i;
-	for (i = 0; i < len; i++) {
-		if (buf[i] != ch) {
-			break;
-		}
-	}
-	return i;
-}
-
 static int getid(char ch) {
 	const char *keys = "[]<>+-,.";
 	const char *cidx = strchr(keys, ch);
 	return cidx ? cidx - keys + 1 : 0;
 }
 
+/* New IL uplift bf */
+#define BF_ADDR_SIZE  64
+#define BF_ALIGN_SIZE 8
+
+struct bf_stack_t {
+	ut64 stack[32];
```

Comment:
Replace `32` with a macro.

Reply:
Agree, something like `BF_STACK_SIZE` and use it later in the code (I spot hardcoded 32 and 31 later). cc @Heersin

Reply:
@Heersin don't forget about this as well, please.